horizons/constants.py

```diff
@@ -246,6 +246,8 @@ class BUILDINGS:
 	GOLD_DEPOSIT     = 91
 	GEM_DEPOSIT      = 92
 
+  AMBIENT          = 72
```

Comment:
Wrong Indentation.

Reply:
Please, use the provided .editorconfig file together with your IDE :). The UH project uses tabs instead of spaces and a couple of other minor tweaks.